frontend/components/App.tsx

```diff
@@ -0,0 +1,45 @@
+export default ({ children }) => (
```

Comment:
@xaun Should I get rid of this global styling method and include a normalize.css instead?